lib_i2s/src/i2s_frame_master_impl.h

```diff
@@ -1,5 +1,6 @@
-// Copyright (c) 2016-2020, XMOS Ltd, All rights reserved
-#if defined(__XS2A__)
+// Copyright (c) 2016-2021, XMOS Ltd, All rights reserved
+// This software is available under the terms provided in LICENSE.txt.
+#if defined(__XS2A__) || defined(__XS3A__)
```

Comment:
I'd future proof this with not defined(__XS1A__)

Reply:
This is better although we have, rightly or wrongly, used "defined(__XS2A__) || defined(__XS3A__)" everywhere else so will keep this for now as we are super short on time.

Reply:
Sure, just thought I'd save you some effort making all the same mods when XS4 comes out ;)